go.mod

```diff
@@ -111,4 +111,4 @@ require (
 	golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c
 )
 
-go 1.14
+go 1.15
```

Comment:
There's no inherent reason we have to do this I guess, but given we're no longer testing with 1.14 this seems reasonable. WDYT @Stebalien?

Reply:
SGTM!